finance-api/user-guides/fetch-account-transactions.md

```diff
@@ -0,0 +1,149 @@
+
+1. Optional: Fetch your organisation UUID. 
+   > **Tip:** You don't need to fetch the organisation UUID, as you can specify the organisation UUID as `self` in requests that require it.
+
+    ```
+    GET users/me
+    ```
+   Example:
+       
+   The following example response returns organisation UUID `ab305d54-75b4-431b-adb2-eb6b9e546013`.
+
+    ```
+    {
+        "uuid": "de305d54-75b4-431b-adb2-eb6b9e546014",
+        "organizationUuid": "ab305d54-75b4-431b-adb2-eb6b9e546013"
+    }
+    ```
+
+2. Fetch all transactions from the merchant's Zettle liquid account.
+      > **Tip:** You specify the organisation UUID as `self`.
+     
+   ```
+   GET /organizations/{organizationUuid}/accounts/liquid/transactions?{start}&{end}
+   ```
+
+   Example:
+   
+   The following example request fetches all transactions from the merchant's Zettle liquid account from 1 January, 2020 to 7 June, 2021.
+   
+   ```
+   GET /organizations/self/accounts/liquid/transactions?start=2021-01-01&end=2021-06-07
+   ```
+       
+   The following example response returns all the transactions from 1 January, 2020 to 7 June, 2021.
+
+    ```json
+    {
+        "data": [
+            {
+                "timestamp": "2021-03-04T04:00:07.285+0000",
+                "amount": -682,
+                "originatorTransactionType": "PAYMENT_FEE",
+                "originatingTransactionUuid": "77db5940-7c1d-11eb-b812-d3f21f3c0d77"
+            },
+            {
+                "timestamp": "2021-03-04T04:00:07.281+0000",
+                "amount": 3300,
+                "originatorTransactionType": "PAYMENT",
+                "originatingTransactionUuid": "77db5940-7c1d-11eb-b812-d3f21f3c0d77"
+            },
+            {
+                "timestamp": "2020-11-21T04:00:15.704+0000",
+                "amount": -621,
+                "originatorTransactionType": "CARD_PAYMENT_FEE",
+                "originatingTransactionUuid": "6820265b-953e-43a7-bb65-abac1ef104bf"
+            },
+            {
+                "timestamp": "2020-11-21T04:00:15.697+0000",
+                "amount": 1100,
+                "originatorTransactionType": "CARD_PAYMENT",
+                "originatingTransactionUuid": "6820265b-953e-43a7-bb65-abac1ef104bf"
+            },
+            {
+                "timestamp": "2020-09-10T09:51:35.162+0000",
+                "amount": 5925,
+                "originatorTransactionType": "FAILED_PAYOUT",
+                "originatingTransactionUuid": "d8550d7a-f347-11ea-9612-3bce5300b9a9"
+            },
+            {
+                "timestamp": "2020-09-10T09:27:28.590+0000",
+                "amount": -5925,
+                "originatorTransactionType": "PAYOUT",
+                "originatingTransactionUuid": "d8550d7a-f347-11ea-9612-3bce5300b9a9"
+            },
+            ...
+        ]
+    }
+    ```
+
+## Step 2: Fetch transactions of certain types during a specific period
```

Comment:
Having this as "step 2" seems that the user _must_ do the previous step in order to make this step works ... I am not sure if they are correlated to be called "steps"... maybe use the term "scenario", "example" or something like this?

Reply:
Totally agree, those shouldn't be steps. They are independent from each other.